.circleci/config.yml

```diff
@@ -45,6 +51,16 @@ reusable-steps:
       paths:
         - ~/.npm
       key: npm-cache-{{ .Revision }}-{{ .BuildNum }}
+  - &save-php-cs-fixer-cache-by-branch
+    save_cache:
+      paths:
+        - .php_cs.cache
+      key: php-cs-fixer-cache-{{ .Branch }}-{{ .BuildNum }}
```

Comment:
are you sure this is correct?
you never restore using `.BuildNum` so it seems pointless.
(i'm aware of how circle ci `restore_cache` works with substrings)

Reply:
i thought you might
save key: 
```
php-cs-fixer-cache-{{ .Branch }}-{{ .Revision }}
```
restore keys:
```
keys:
  - php-cs-fixer-cache-{{ .Branch }}-{{ .Revision }}
  - php-cs-fixer-cache-{{ .Branch }}
  - php-cs-fixer-cache
```

Reply:
It's intentional. The .BuildNum is for cache busting.

Reply:
ah, in my project i actually generate a hash of all php files they php-cs-fixer runs over, then cache on that.

something like:
`find src tests -type f -name "*.php" -exec md5sum {} + > php-cs-fixer.checksum`

```
- restore_cache:
  keys:
    - php-cs-fixer-{{ checksum "php-cs-fixer.checksum" }}
    - php-cs-fixer-
```

then there is no need for cache busting.